dev-env/windows/libexec/core.ps1

```diff
@@ -43,14 +43,15 @@ function da_auto_prevent_update {
 }
 
 function da_clear_buckets {
-    Remove-Item $scoopInstallDir\apps\scoop\current\bucket\* -r -force
+    Remove-Item $scoopInstallDir\buckets\dadew -r -force -ErrorAction Ignore
+    New-Item $scoopInstallDir\buckets\dadew -ItemType "directory" -Force
 }
 
 function da_sync_buckets([String] $Directory) {
     da_clear_buckets
     $files = Get-ChildItem $Directory | Where-Object {$_.Name -like '*.json'}
     ForEach ($file in $files) {
-        Get-Content "$Directory\$file" | Set-Content "$scoopInstallDir\apps\scoop\current\bucket\$file"
+        Get-Content "$Directory\$file" | Set-Content "$scoopInstallDir\buckets\dadew\$file"
```

Comment:
I couldn’t actually find any docs for where you need to put the manifest files but the previous location no longer seems to work whereas this one does :shrug: